app/main/posts/common/post-actions.directive.js

```diff
@@ -71,7 +71,7 @@ function PostActionsDirective(
                 return;
             }
 
-            if ($routeParams.view !== 'data' && $location.path().indexOf('data') === -1) {
+            if ($stateParams.view !== 'data' && $location.path().indexOf('data') === -1) {
```

Comment:
Shouldn't that be `$uiRouterGlobals.params` not `$stateParams` ? (or transition params, I'm not sure)